server/src/main/java/org/apache/druid/query/lookup/LookupSerdeModule.java

```diff
@@ -58,6 +59,7 @@ public List<? extends Module> getJacksonModules()
   public void configure(Binder binder)
   {
     JsonConfigProvider.bind(binder, LookupModule.PROPERTY_BASE, LookupConfig.class);
+    binder.bind(LookupNodeService.class).toInstance(new LookupNodeService(LookupListeningAnnouncerConfig.DEFAULT_TIER));
```

Comment:
Please add a comment here on why this needs to be done.

Reply:
added